### PR TITLE
DOCS-#7: Add deps for building docs

### PR DESCRIPTION
**Pull request by YarShev**
Signed-off-by: Igoshev, Yaroslav <yaroslav.igoshev@intel.com>

### CODE REVIEW

**Comment by YarShev** [comment]
@no-ponomarev , @prutskov , do you think it would good if we place all deps (related to tests run for all backends and for building docs) for development in one place (environment.yml and requirements.txt)?

**Comment by prutskov** [comment]
> @no-ponomarev , @prutskov , do you think it would good if we place all deps (related to tests run for all backends and for building docs) for development in one place (environment.yml and requirements.txt)?

I like this approach. But it would be good to highlight  section of deps related to docs in requirements.txt/environment.yml.

**Comment by YarShev** [comment]
> I like this approach. But it would be good to highlight section of deps related to docs in requirements.txt/environment.yml.

We can add a comment for dev deps and another one for docs deps.

**Comment by YarShev** [comment]
@no-ponomarev , if you are okay with this, I'll add the deps for building docs to env files after we merge #23.

File: docs/conf.py [hunk 1]
```diff
@@ -14,6 +14,12 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
```
Comment (no-ponomarev) [inline docs/conf.py:15]: Maybe its time to remove this?

Comment (YarShev) [reply docs/conf.py:15]: good point

Comment (no-ponomarev) [inline docs/conf.py:15]:
```suggestion
```

Comment (YarShev) [reply docs/conf.py]: fixed

Comment (YarShev) [reply docs/conf.py]: fixed

**Pull request merged**